ray-operator/controllers/ray/common/pod.go

```diff
@@ -513,6 +513,13 @@ func setContainerEnvVars(pod *v1.Pod, rayContainerIndex int, rayNodeType rayiov1
 		portEnv := v1.EnvVar{Name: RAY_PORT, Value: headPort}
 		container.Env = append(container.Env, portEnv)
 	}
+	if strings.ToLower(creator) == RayServiceCreatorLabelValue {
+		// Only add this env for Ray Service cluster to improve service SLA.
```

Comment:
do we only need this in Ray Service? what about other scenarios?

Reply:
@iycheng mentioned this will increase SLA for ray service.
But for ray job, it will ignore the error info which is not good for observability. So for now, we keep it only for RayService.